ros/finalarm_simplified_control/launch/finalarm_gazebo_control.launch

```diff
@@ -1,22 +1,20 @@
 <launch>
-  <!-- Load joint controller configurations from YAML file to parameter server -->
-  <rosparam file="$(find finalarm_simplified_control)/config/finalarm_gazebo_control.yaml" command="load"/>
-
-  <!-- load the controllers -->
-  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
-    output="screen" ns="/finalarm_simplified" args=" joint_state_controller
-                        joint_1_position_controller
-                        joint_2_position_controller
-                        joint_3_position_controller
-                        joint_4_position_controller
-                        joint_5_position_controller
-                        joint_6_position_controller
-                        joint_7_position_controller"/>
-
-  <!-- convert joint states to TF transforms for rviz, etc -->
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
-    respawn="false" output="screen">
-    <remap from="/joint_states" to="/finalarm_simplified/joint_states" />
-  </node>
-
+  <!-- NOTE: this is the same as finalarm except without motors 8 and 9 -->
+  <include
+    file="$(find finalarm_control)/launch/_spawn_gazebo_controllers.launch"
+  >
+    <arg
+      name="spawner_args"
+      value="
+        finalarm_position_controller_1
```

Comment:
The only thing about these is that they don't follow the convention of using `finalarm_simplified` as the model name. We don't use namespacing of the topics anywhere else, so this would not work in gazebo and on the real robot the same way. Arguable how this should be implemented.